_drafts/2020-12-14-data_science_interview_prep_data_manipulation.md

```diff
@@ -0,0 +1,370 @@
+---
+layout: post
+title: "Data Science Interview Practice: Data Manipulation"
+description: >
+image: /files/switrs-dataset/tram_auto_crash_in_1957_frederiksplein_amsterdam.jpg
+show_lead_image: True
+image_alt: >
+categories:
+  - career-advice
+  - interview-prep
+---
+
+<!-- Simple script to call from the "Show/Hide" buttons-->
+<script type="text/javascript">
+function showhide(d) {
+  d.style.display = (d.style.display !== "none") ? "none" : "block";
+}
+</script>
+
+{% assign twitter-name = site.author.twitter %}
```

Comment:
This seems like it should be in the header or template or something. Do you have to spell it out in every post?

Reply:
Actually I should just use site.author.twitter... No reason to alias it.